parts/skip-nav.html

```diff
@@ -1,13 +1,15 @@
-<!-- wp:buttons {"className":"skip-nav-menu screenreader-text","layout":{"type":"flex","justifyContent":"center"}} -->
-<div class="wp-block-buttons skip-nav-menu screenreader-text"><!-- wp:button {"className":"is-style-outline skip-nav"} -->
-    <div class="wp-block-button is-style-outline skip-nav"><a class="wp-block-button__link" href="#post-content">Skip to Main Content</a></div>
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"center"}} -->
+<div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:buttons {"className":"skip-nav-menu screenreader-text","layout":{"type":"flex","justifyContent":"center"}} -->
+    <div class="wp-block-buttons skip-nav-menu screenreader-text"><!-- wp:button {"backgroundColor":"background","textColor":"headline-color","className":"skip-nav is-style-fill","iconsList":[{"label":"None","value":""},{"label":"Default","value":"default"},{"label":"Home","value":"home"},{"label":"Auto","value":"auto"},{"label":"Book","value":"book"},{"label":"Calendar","value":"calendar"},{"label":"Case","value":"case"},{"label":"Changes","value":"changes"},{"label":"Federal","value":"federal"},{"label":"Gear","value":"gear"},{"label":"Group","value":"group"},{"label":"Leaf","value":"leaf"},{"label":"Map","value":"map"},{"label":"Tag","value":"tag"}]} -->
+    <div class="wp-block-button skip-nav is-style-fill"><a class="wp-block-button__link has-headline-color-color has-background-background-color has-text-color has-background wp-element-button" href="#post-content">Skip to Main Content</a></div>
     <!-- /wp:button -->
     
-    <!-- wp:button {"className":"is-style-outline skip-nav"} -->
-    <div class="wp-block-button is-style-outline skip-nav"><a class="wp-block-button__link" href="#main-menu">Skip to Navigation</a></div>
+    <!-- wp:button {"backgroundColor":"background","textColor":"headline-color","className":"skip-nav is-style-fill","iconsList":[{"label":"None","value":""},{"label":"Default","value":"default"},{"label":"Home","value":"home"},{"label":"Auto","value":"auto"},{"label":"Book","value":"book"},{"label":"Calendar","value":"calendar"},{"label":"Case","value":"case"},{"label":"Changes","value":"changes"},{"label":"Federal","value":"federal"},{"label":"Gear","value":"gear"},{"label":"Group","value":"group"},{"label":"Leaf","value":"leaf"},{"label":"Map","value":"map"},{"label":"Tag","value":"tag"}]} -->
```

Comment:
just wondering what the different labels here are for, ie default, auto, calendar, changes, gear, leaf, tag.

Reply:
It's meta from the icon block style filter.